app/src/main/java/org/mifos/mobile/ui/activities/base/BaseActivity.java

```diff
@@ -214,15 +215,29 @@ public void replaceFragment(Fragment fragment, boolean addToBackStack, int conta
 
         if (!fragmentPopped && getSupportFragmentManager().findFragmentByTag(backStateName) ==
                 null) {
+            Fragment previousFragment = getSupportFragmentManager().findFragmentById(containerId);
             FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+            transition(previousFragment, fragment);
             transaction.replace(containerId, fragment, backStateName);
             if (addToBackStack) {
                 transaction.addToBackStack(backStateName);
             }
-            transaction.commit();
+            transaction.commitAllowingStateLoss();
         }
     }
 
+    public void transition(Fragment previousFragment, Fragment nextFragment){
+        if(previousFragment!=null) {
+            Fade exitFade = new Fade();
+            exitFade.setDuration(250);
```

Comment:
Instead of using hardcoded values, what about if we use standard timing for animations suggested by Googlers like this:
`exitFade.setDuration(android.R.integer.config_shortAnimTime)`